### PR TITLE
Add AppFlowy 0.0.2

### PR DESCRIPTION
**Pull request by stephengroat**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/appflowy.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask "appflowy" do
+  version "0.0.2"
+  sha256 "3d5177015cc568eab83a2e1d4ff71fe28ad31d63834dc3aa32c23742b09b6b29"
+
+  url "https://github.com/AppFlowy-IO/appflowy/releases/download/#{version}/AppFlowy-MacOS.zip",
+      verified: "github.com/AppFlowy-IO/appflowy/"
+  name "appflowy"
+  desc "Open-source alternative to Notion"
```
Comment (bevanjkay) [inline Casks/appflowy.rb:8]: @stephengroat The `desc` should describe what the software does (including for somebody who doesn't know what Notion is). See the [`desc` docs](https://docs.brew.sh/Cask-Cookbook#stanza-desc) for examples.

**Review by bevanjkay — CHANGES_REQUESTED** [review]
@stephengroat Thanks for your contribution.
Please try to add a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

File: Casks/appflowy.rb (hunk 1 above)
Comment (stephengroat) [reply Casks/appflowy.rb]: addressed in 562be83

**Comment by stephengroat** [comment]
> @stephengroat Thanks for your contribution. Please try to add a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

addressed in 562be83

**Review by bevanjkay — DISMISSED** [review]

**Comment by bevanjkay** [comment]
Thank you @stephengroat

**Pull request merged**